apps/blog/src/layouts/base.astro

```diff
@@ -59,7 +59,7 @@ if (theme.fonts.font_family.secondary) {
   </head>
   <body class="bg-body font-primary font-normal leading-relaxed text-text">
     <Header />
-    <main class="min-h-[60vh]">
+    <main class="min-h-[60vh] max-w-5xl px-8 mx-auto">
```

Comment:
In each section there should be a container defined. This will have bunch of issues in the future. 

example: post-base.astro:13